Makefile

```diff
@@ -67,8 +67,14 @@ local:
 	@rm -rf /var/tmp/$(PKGNAME)-$(VERSION)
 	@echo "The archive is in $(PKGNAME)-$(VERSION).tar.gz"
 
+test-container:
+	sudo docker build -t larskarlitski/lorax-tests:latest -f Dockerfile.test .
+
+check-in-docker:
+	sudo docker run --rm -ti -v `pwd`:/lorax larskarlitski/lorax-tests make check
```

Comment:
One of the problems with mounting this is that you end up with root owned artifacts in your home directory when running this locally. You also end up with SELinux problems unless you add `--security-opt label=disable`

Reply:
Oh, good point. I've managed to make the mount read-only for `make check`, but not for `make dist`. The tests expect to be able to read and write from several directories in the source tree. Do you know if that's easily fixable?

Reply:
Probably the document generation. It expects to be able to generate the docs so it can test serving them. Maybe we could drop that bit.